tests/check_consistent.py

```diff
@@ -140,11 +141,21 @@ def get_txt_requirements() -> dict[str, SpecifierSet]:
         return {requirement.name: requirement.specifier for requirement in requirements}
 
 
+class PreCommitConfigRepos(TypedDict):
+    hooks: list[dict[str, str]]
+    repo: str
+    rev: str
+
+
+class PreCommitConfig(TypedDict):
+    repos: list[PreCommitConfigRepos]
+
+
 def get_precommit_requirements() -> dict[str, SpecifierSet]:
     with open(".pre-commit-config.yaml", encoding="UTF-8") as precommit_file:
         precommit = precommit_file.read()
-    yam = yaml.load(precommit, Loader=yaml.Loader)
-    precommit_requirements = {}
+    yam: PreCommitConfig = yaml.load(precommit, Loader=yaml.Loader)
+    precommit_requirements = dict[str, SpecifierSet]()
```

Comment:
```suggestion
    precommit_requirements: dict[str, SpecifierSet] = {}
```
Nit, but would prefer this as it separates the types more explicitly from the runtime.

Reply:
Same, I thought I had changed them all (except `set`) to separate typing and assignment. Missed that one thanks!